### PR TITLE
Guild level progress dashboard, and cleanup of requirements 

### PR DESCRIPTION
**Pull request by Corvux89**
(hopefully I did the recompile correctly)

### CODE REVIEW

File: ProphetBot/cogs/admin.py [hunk 1]
```diff
@@ -148,9 +150,30 @@ async def list(self, ctx: ApplicationContext):
         name="test"
     )
     async def test(self, ctx: ApplicationContext):
```
Comment (ShardlessBun) [inline ProphetBot/cogs/admin.py:152]: Not opposed to having test functionality left in, but this method should have a more descriptive name (and/or a quick comment saying what it currently does)

File: requirements.txt [hunk 2]
```diff
@@ -4,108 +4,14 @@
...
-    #   marshmallow
-pep517==0.12.0
-    # via build
-pip-tools==6.12.1
-    # via -r requirements.in
-psycopg2-binary==2.9.5
-    # via
-    #   aiopg
-    #   sqlalchemy
-py-cord==2.2.2
```
Comment (ShardlessBun) [inline requirements.txt:75]: Something definitely went wrong with this requirements compilation. Several packages from requirements.in are marked as deleted, and it looks like dependencies aren't being pulled in.

Honestly, the fastest way to resolve this is probably to delete the entire requirements.txt file, then run `pip-compile requirements.in` to get a completely fresh set of requirements. Versions may jump up a bit, but I'm not too worried about that.

File: ProphetBot/models/embeds/ref_embeds.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
+import io
```
Comment (ShardlessBun) [inline ProphetBot/models/embeds/ref_embeds.py:1]: I don't see this new package used anywhere in this file. Is the import necessary?

File: ProphetBot/helpers/entity_helpers.py [hunk 4]
```diff
@@ -3,6 +3,7 @@
 import random
 from datetime import datetime
 from statistics import mean
+from typing import List
```
Comment (ShardlessBun) [inline ProphetBot/helpers/entity_helpers.py:6]: Looks like this import isn't used. Should probably delete it if that's the case.

**Review by ShardlessBun — CHANGES_REQUESTED** [review]
Let me know if you have any questions. The requirements issue is the big one since this will 100% not run in its current state.

File: ProphetBot/cogs/admin.py (hunk 1 above)
Comment (Corvux89) [reply ProphetBot/cogs/admin.py]: Removed test command

**Pull request merged**